index.js

```diff
@@ -39,37 +24,48 @@ module.exports = {
     }
     this.app = app;
 
-    for (let i = 0; i < FONT_FILES.length; i++) {
-      app.import(fontPath(app, FONT_FILES[i]), {
-        destDir: 'assets'
-      });
-    }
-
     if (!(app.options['materialize-shim'] || {}).omitJS) {
       app.import('vendor/materialize/materialize.js');
-      app.import('vendor/materialize-shim.js', {
-        exports: {
```

Comment:
(for my own reference) I just found the [docs that mention this](https://guides.emberjs.com/v2.18.0/addons-and-dependencies/managing-dependencies/#toc_amd-javascript-modules), but it's still not clear to me why/when one would want to use this. Also, I would expect this to be defined somewhere in the `ember-cli` docs, but [this is the closest thing I've found](https://ember-cli.com/api/classes/EmberApp.html#method_import).